std_msgs/Header header
bool is_enabled
int32 pointed_id      # An invalid id means no item is pointed
int32[] selected_ids